src/main/java/PalindromeChecker.java

```diff
@@ -3,11 +3,14 @@
 
     public boolean check(String word) {
 
+        //still needed?
```

Comment:
I suspect you could remove these codes that check for a single character word and everything would still work.